### DISCUSSION TITLE
Unused variables

### DISCUSSION THREAD

**Discussion by mburns08109** [twinbasic/twinbasic#443] [Ideas]
Can we get tB to throw a compiler warning if it detects declared, but unused variables?

**Comment by WaynePhillipsEA** [comment]
Yes, we should do that.  I've added this request to the compiler warnings issue #208.  Thanks

**Reply by YogiYang** [reply]
It would be great if you could do the same thing for Functions and Subs in user created modules.
But for this to happen you will need to find a way to distinguish modules that are provided with tB and those created by users.
